packages/manager/modules/pci/src/projects/project/instances/instances.controller.js

```diff
@@ -51,6 +51,14 @@ export default class CloudProjectComputeInfrastructureListCtrl {
     );
   }
 
+  getBadgeTooltipText(status) {
+    return status === 'STOPPED' || status === 'SUSPENDED'
+      ? this.$translate.instant(
+          `pci_projects_project_instances_status_${status.toUpperCase()}_TOOLTIP`,
```

Comment:
```suggestion
          `pci_projects_project_instances_status_${status}_TOOLTIP`,
```
As you compare status to strings in upper case, it means status is already in upper case format